decksite/main.py

```diff
@@ -144,13 +144,36 @@ def archetypes() -> str:
     view = Archetypes(deckless_archetypes, all_matchups)
     return view.page()
 
+@APP.route('/archetypes/tournament/')
```

Comment:
Alternative approach is to load everything into /archetypes/ and flip between the two with js or something. Given page size concerns and addressibilty I think I like the way you've gone here but could be swayed.

Reply:
Honestly, I think that it would be nicer to switch like that, but I don't know how to do the JS. Maybe I'll revisit that in the future.